src/components/Calculator.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="main-container">
+    <input type="text" className="grid-box display" value="0" />
+    <button type="button" className="grid-box">
+      AC
+    </button>
+    <button type="button" className="grid-box">
+      +/-
+    </button>
+    <button type="button" className="grid-box">
+      %
+    </button>
+    <button type="button" className="grid-box left">
+      ÷
+    </button>
+    <button type="button" className="grid-box">
+      7
+    </button>
+    <button type="button" className="grid-box">
+      8
+    </button>
+    <button type="button" className="grid-box">
+      9
+    </button>
+    <button type="button" className="grid-box left">
+      x
+    </button>
+    <button type="button" className="grid-box">
+      4
+    </button>
+    <button type="button" className="grid-box">
+      5
+    </button>
+    <button type="button" className="grid-box">
+      6
+    </button>
+    <button type="button" className="grid-box left">
+      -
+    </button>
+    <button type="button" className="grid-box">
+      1
+    </button>
+    <button type="button" className="grid-box">
+      2
+    </button>
+    <button type="button" className="grid-box">
+      3
+    </button>
+    <button type="button" className="grid-box left">
+      +
+    </button>
+    <button type="button" className="grid-box btn-zero">
+      0
+    </button>
+    <button type="button" className="grid-box">
+      .
+    </button>
+    <button type="button" className="grid-box left">
+      =
+    </button>
+  </div>
+);
```

Comment:
- [ ] well done starting this project correctly, according to the project requirements, the Calculator.js component should at least have one child component, this is a way to teach you about building reusable components in React, kindly make those buttons a component and import it here :+1:

Reply:
Okay👍